include/tvm/runtime/container.h

```diff
@@ -198,6 +198,7 @@ class ADTObj : public Object, public InplaceArrayBase<ADTObj, ObjectRef> {
   }
 
   friend class ADT;
+  template <typename ArrayType, typename ElemType>
```

Comment:
consider change to InplaceArrayBase<ADTObj, ObjectRef>

Reply:
Thanks!  Changed it.  This compiles clean here (VS 2019 toolset v14.2)